airflow/www/static/css/calendar.css

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+text.year-label {
+  font-size: 16px;
+  font-weight: bold;
+}
+
+text.day-label {
+  font-size: 12px;
+}
+
+path.month {
+  stroke: black;
+  stroke-width: 2px;
+}
+
+rect.day {
+  stroke: #ccc;
+  stroke-width: 1px;
+  cursor: pointer;
+}
+
+rect.success {
+  fill: green;
+}
+
+rect.running {
+  fill: lime;
+}
+
+rect.failed {
+  fill: red;
+}
+
+rect.no_status {
+  fill: #eee;
```

Comment:
No Status is white elsewhere in the app. We should stay consistent here too.

Reply:
Thanks, I have fixed it